Gemfile

```diff
@@ -3,6 +3,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.1.1'
 
+gem 'font-awesome-rails'
+
 gem 'erb-formatter'
 
 gem 'rubocop', '>= 1.0', '< 2.0'
```

Comment:
- [ ] Kindly add `gem 'ffi'` to your Gemfile for windows users to successfully run your tests without errors 😄